frontend/src/__tests__/cypress/cypress/fixtures/resources/yaml/persistentVolumeClaim.yaml

```diff
@@ -0,0 +1,22 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{PVC_NAME}}
+  namespace: {{NAMESPACE}}
+  labels:
+    opendatahub.io/dashboard: 'true'
+  annotations:
+    openshift.io/description: ''
+    openshift.io/display-name: {{PVC_DISPLAY_NAME}}
+  finalizers:
+    - kubernetes.io/pvc-protection
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
```

Comment:
@FedeAlonso What about the PV size, can you pass it to the template too ?

Reply:
Done in 5792daa
![image](https://github.com/user-attachments/assets/e4d646c1-d54d-4e19-8446-fdcc72b54dcf)